### ISSUE TITLE
zigbee2mqtt.io breaks DNSSEC

### ISSUE THREAD

**Issue by electrofloat** [Koenkk/zigbee2mqtt#2012]
Since there's no issue tracking on zigbee2mqtt.io repo I had to create this one here.
I noticed that the website is regularly unavailable, and it seemed random.  Investigation showed that the issue is with the CNAME record at apex.
Such zones are **illegal** according to the RFCs:
 - https://tools.ietf.org/html/rfc1034#section-3.6.2
 - https://tools.ietf.org/html/rfc2181#section-10.1

http://dnsviz.net/d/zigbee2mqtt.io/dnssec/

Can you please get in contact with your DNS provider and make them fix this issue asap?

Thank you!

**Comment by Koenkk** [comment]
The current configuration is:

![image](https://user-images.githubusercontent.com/2892853/65393221-6284d100-dd7e-11e9-8d0e-8006f21776e8.png)

I've changed it to

![image](https://user-images.githubusercontent.com/2892853/65393223-67498500-dd7e-11e9-826e-7c4c31612aa9.png)

Does that fix this issue?

**Comment by electrofloat** [comment]
Yes it does. Although it wasn't your fault in the first place, it is a good fix for the mentioned issue.

Dynadot shouldn't let their customers configure an illegal DNS record at all. Even if you fixed it for yours, any other domains they own could have this exact issue.
So I'm still encouraging you to let them know about this, and hopefully they'll implement some sort of validation or not allow this kind of configuration at all for any of their domains.

Thank you!

**Comment by Koenkk** [comment]
I've send them a mail, thanks!
